packages/sui-pde/src/hooks/useExperiment.js

```diff
@@ -8,14 +8,16 @@ import {getPlatformStrategy} from './common/platformStrategies.js'
  * @param {string} param.experimentName
  * @param {object} param.attributes
  * @param {function} param.trackExperimentViewed
- * @param {string} [param.queryString] Test purposes only
+ * @param {string} param.queryString
+ * @param {string} param.adapterId Adapter id to be executed
  * @return {{variation: string}}
  */
 export default function useExperiment({
   experimentName,
   attributes,
   trackExperimentViewed,
-  queryString // for test purposes only
+  queryString,
+  adapterId
```

Comment:
Is `adapterId ` required?
Could you add some examples about useFeature and useExperiment when using multiple adapters?

Reply:
done